Framework/PythonInterface/plugins/algorithms/LoadDNSLegacy.py

```diff
@@ -132,6 +136,13 @@ def PyExec(self):
             self.log().error(message)
             raise RuntimeError(message)
 
+        if wavelength < 0.01:
+            wavelength = metadata.wavelength
+
+        # calculate incident energy, since given in the data file is wrong
+        velocity = h/(m_n*wavelength*1e-10)   # m/s
+        incident_energy = 0.5*m_n*velocity*velocity/1.602176487e-22  # meV
```

Comment:
One could use `scipy.constants` and `velocity**2` here.

Reply:
x*x is about 1.5 times faster, than x**2. It is not critical for this algorithm, but I'd prefer leave velocity*velocity.